modules/swagger-codegen-maven-plugin/examples/java-client.xml

```diff
@@ -12,7 +12,7 @@
             <plugin>
                 <groupId>io.swagger</groupId>
                 <artifactId>swagger-codegen-maven-plugin</artifactId>
-                <version>2.1.5-SNAPSHOT</version>
+                <version>2.2.0-SNAPSHOT</version>
```

Comment:
2.2.0 has been released: https://mvnrepository.com/artifact/io.swagger/swagger-codegen-maven-plugin

Please kindly remove the `-SNAPSHOT`